Sources/Base/Dispatch.swift

```diff
@@ -1,5 +1,15 @@
+#if os(Linux)
 import Strand
 
 public func background(function: () -> Void) throws {
     let _ = try Strand(closure: function)
 }
+#else
+import Foundation
+
+let background = DispatchQueue.global()
```

Comment:
I believe so, unfortunately, I can't find good documentation on new Dispatch APIs so it's more difficult for me to optimize completely.